### DISCUSSION TITLE
Azure AD Auth:  Unauthorized

### DISCUSSION THREAD

**Discussion by copenhaus** [requarks/wiki#6555] [Error / Bug Report]
Hi.  Setting up my first Wiki.js server as a Docker container.

I want to set it up so the user can authenticate against Azure AD.  I've read the [doc](https://docs.requarks.io/auth/azure) and followed it the best I know how.

I've entered:
- Callback URL / Redirect URI 
- Application (Client) ID
- OpenID Connect metadata document
- entered Logout URL and checked ID tokens (used for implicit and hybrid flows)

But on the home page, either button gave me "Unauthorized"  "You cannot view this page."

Despite the error message and cannot login, I do see the user added in Users, with Provider as "Azure Active Directory"

I've searched this discussion group but no one seemed to have reported this.

Thoughts/advice is greatly appreciated.

**Comment by Idan17** [comment]
Hi, 
have you ever managed to solve this problem?
i'm facing the same with my wiki.js installation

**Reply by aridwan** [reply]
try to allow self registration on your wiki.js azure settings.

**Comment by Te-k** [comment]
Hi everyone,

I am facing the same issue with my wikijs installation, even with self registration enabled. Is that a known issue? Has anyone used Azure AD auth successfully?
